share/haarDetection/CMakeLists.txt

```diff
@@ -6,6 +6,21 @@ yarp_install(FILES conf/haarDetection.ini
                    conf/haarDetection2D.ini
                    conf/haarcascade_cocacola_can.xml
                    conf/haarcascade_frontalface_alt.xml
+		   conf/haarcascade_car.xml  
+		   conf/haarcascade_coke.xml       
+		   conf/haarcascade_lemon.xml  
+		   conf/haarcascade_sofa.xml
+		   conf/haarcascade_chair.xml  
+		   conf/haarcascade_clock.xml    
+		   conf/haarcascade_cup.xml  
+		   conf/haarcascade_mouse.xml  
+		   conf/haarcascade_table.xml  
+		   conf/haarcascade_cutlery.xml  
+		   conf/haarcascade_keyboard.xml 
+		   conf/haarcascade_pal_logo.xml   
+		   conf/haarcascade_tv.xml  
+		   conf/haarcascade_dog.xml  
+		   conf/haarcascade_lamp.xml  
```

Comment:
Please use spaces instead of tabs.

Reply:
Done at 5d02f26a5637680c8c4c557a832eb1d8251448fa.